core/src/validator.rs

```diff
@@ -868,10 +868,10 @@ impl TestValidator {
             42,
             bootstrap_validator_lamports,
         );
-        genesis_config
-            .native_instruction_processors
-            .push(solana_bpf_loader_program!());
-
+        // TODO
```

Comment:
@mvines Not sure how to prime the test validator with the bpf_loader.  Is there some way to access the bank?

Reply:
There's no way to get at the bank at this level, it doesn't even exist yet..  

Passing a `Vec<Program>` into `Validator::new()` could be one way, somewhat like what you have commented out.   It's not obvious to me why the normal `solana_genesis_programs::get_programs(0)` path doesn't work here though.